lib/events/filesessions/fileasync.go

```diff
@@ -244,14 +259,16 @@ func (u *Uploader) Scan(ctx context.Context) (*ScanStats, error) {
 			}
 			if isSessionError(err) {
 				u.log.WithError(err).Warningf("Skipped session recording %v.", fi.Name())
+				stats.Corrupted++
 				continue
 			}
 			return nil, trace.Wrap(err)
 		}
 		stats.Started++
 	}
 	if stats.Scanned > 0 {
-		u.log.Debugf("Scanned %v uploads, started %v in %v.", stats.Scanned, stats.Started, u.cfg.ScanDir)
+		u.log.Debugf("Scanned %v uploads, started %v, found %v corrupted in %v.",
```

Comment:
Maybe log corrupted stats only it if we've got at least one? I assume these are pretty rare.

Reply:
I don't think it hurts anything including it here. Another if statement inside here is just extra branching for little benefit.